### PR TITLE
Move 12 

### PR DESCRIPTION
**Pull request by remmi755**
Clear A7

### CODE REVIEW

**Comment by mentor-tara** [comment]
### Oh no! You hit a mine. Game over :slightly_frowning_face:

This is the current board:

|   | **A** | **B** | **C** | **D** | **E** | **F** | **G** | **H** |
|---|:---:|:---:|:---:|:---:|:---:|:---:|:---:|:---:|
| **1** |   |   |   |   |   |   |   |   |
| **2** |   |   |   |   |   |   |   |   |
| **3** |   |   |   |   |   |   |   |   |
| **4** |   |   |   |   |   |   |   |   |
| **5** |   |   |   |   |   |   |   |   |
| **6** |   |   |   |   |   |   |   |   |
| **7** | :bomb: |   |   |   |   |   |   |   |
| **8** | :pirate_flag: |   |   |   |   |   |   |   |

And this is the revealed board:

|   | **A** | **B** | **C** | **D** | **E** | **F** | **G** | **H** |
|---|:---:|:---:|:---:|:---:|:---:|:---:|:---:|:---:|
| **1** | 0 | 0 | 0 | 1 | :bomb: | 2 | :bomb: | 2 |
| **2** | 0 | 0 | 0 | 1 | 1 | 2 | 2 | :bomb: |
| **3** | 0 | 0 | 0 | 0 | 0 | 0 | 1 | 1 |
| **4** | 0 | 0 | 0 | 0 | 0 | 0 | 1 | 1 |
| **5** | 1 | 1 | 0 | 1 | 1 | 1 | 1 | :bomb: |
| **6** | :bomb: | 3 | 1 | 1 | :bomb: | 2 | 2 | 2 |
| **7** | :bomb: | :bomb: | 1 | 1 | 1 | 2 | :bomb: | 1 |
| **8** | :bomb: | 3 | 1 | 0 | 0 | 1 | 1 | 1 |

---

### This game is over but I created a PR so you can start a new game. You can close this PR.[](clear-A7)

File: commands.json [hunk 1]
```diff
@@ -1,6 +1,7 @@
 {
   "commands": [
```
Comment (remmi755) [inline commands.json:2]: hello

**Review by remmi755 — COMMENTED** [review]
thanks

File: commands.json [hunk 2]
```diff
@@ -1,6 +1,7 @@
...
-"flag A8"
+"flag A8",
+    "clear A7"
```
Comment (remmi755) [inline commands.json:4]: ff

**Review by remmi755 — COMMENTED** [review]
ffffff

**Pull request closed without merging**